src/pages/home/HeaderView.js

```diff
@@ -308,6 +325,25 @@ function HeaderView(props) {
                                                 </Text>
                                             </PressableWithoutFeedback>
                                         )}
+                                        {isPolicyExpenseChat && !_.isEmpty(policyDescription) && _.isEmpty(parentNavigationSubtitleData) && (
+                                            <PressableWithoutFeedback
+                                                onPress={() => {
+                                                    if (!ReportUtils.canEditPolicyDescription(props.policy)) {
+                                                        return;
+                                                    }
+                                                    Navigation.navigate(ROUTES.WORKSPACE_DESCRIPTION.getRoute(props.report.policyID));
```

Comment:
```suggestion
                                                    if (ReportUtils.canEditPolicyDescription(props.policy)) {
                                                                                                            Navigation.navigate(ROUTES.WORKSPACE_DESCRIPTION.getRoute(props.report.policyID));
                                                        return;
                                                    }
                                                    Navigation.navigate(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(props.reportID));
```

I think this would work fine. @esh-g

Reply:
Thanks for that! No need to overcomplicate with the nested touch stuff